src/manage.c

```diff
@@ -4465,110 +4516,104 @@ manage_system_report (const char *name, const char *duration,
-      GString *buffer;
-
-      g_debug ("%s: gvmcg failed with %d", __func__, exit_status);
-      g_debug ("%s: stdout: %s", __func__, astdout);
-      g_debug ("%s: stderr: %s", __func__, astderr);
-      g_free (astdout);
-      g_free (astderr);
-      g_free (command);
-
-      buffer = g_string_new (FALLBACK_SYSTEM_REPORT_HEADER);
-
-      ret = getloadavg (load, 3);
-      if (ret == 3)
+      if (!g_find_program_in_path ("gvmcg"))
         {
-          g_string_append_printf (buffer,
-                                  "Load average for past minute:     %.1f\n",
-                                  load[0]);
-          g_string_append_printf (buffer,
-                                  "Load average for past 5 minutes:  %.1f\n",
-                                  load[1]);
-          g_string_append_printf (buffer,
-                                  "Load average for past 15 minutes: %.1f\n",
-                                  load[2]);
+          return_code = 7;
         }
-      else
-        g_string_append (buffer, "Error getting load averages.\n");
-
-      get_error = NULL;
-      g_file_get_contents ("/proc/meminfo",
-                           &output,
-                           &output_len,
-                           &get_error);
-      if (get_error)
-        g_error_free (get_error);
       else
         {
-          gchar *safe;
-          g_string_append (buffer, "\n/proc/meminfo:\n\n");
-          safe = g_markup_escape_text (output, strlen (output));
-          g_free (output);
-          g_string_append (buffer, safe);
-          g_free (safe);
-        }
+          /* For simplicity, it's up to the command to do the base64
+           * encoding.
+           */
+          if (params_count == 1)
+            command = g_strdup_printf ("gvmcg %ld %s",
+                                       cmd_param_1,
+                                       name);
+          else
+            command = g_strdup_printf ("gvmcg %ld %ld %s",
+                                       cmd_param_1,
+                                       cmd_param_2,
+                                       name);
+
+          g_debug ("   command: %s", command);
+
+          if ((g_spawn_command_line_sync (command,
+                                          &astdout,
+                                          &astderr,
+                                          &exit_status,
+                                          &err)
+               == FALSE)
+              || (WIFEXITED (exit_status) == 0)
+              || WEXITSTATUS (exit_status))
+            {
+              return_code = 3;
 
+              g_warning ("%s: Failed to create performance graph -- %s",
+                         __func__, astderr);
+              g_debug ("%s: gvmcg failed with %d", __func__, exit_status);
+              g_debug ("%s: stdout: %s", __func__, astdout);
+              g_debug ("%s: stderr: %s", __func__, astderr);
+            }
+          g_free (command);
+	}
```

Comment:
The indentation should use only spaces:
```suggestion
        }
```